source/fuzz/fact_manager.h

```diff
@@ -190,6 +194,15 @@ class FactManager {
   // End of irrelevant pointee value facts
   //==============================
 
+  //==============================
+  // Querying facts about irrelevant ids.
```

Comment:
Given that we have merged the sub-managers for these facts and the above facts, I think it would make sense to combine their "Querying facts about ..." comment blocks; I guess "Querying facts about irrelevant values" would be good.

Reply:
Fixed in 9a66484.